enhancements/machine-config/bootc_update_path.md

```diff
@@ -0,0 +1,185 @@
+---
+title: bootc-update-path
+authors:
+  - "@umohnani8"
+  - "@inesqyx"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@cheesesashimi"
+  - "@mrunalp"
+  - "@cgwalters"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@sinnykumari"
+  - "@yuqi-zhang"
+  - "@mrunalp"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: 2024-05-28
+last-updated: 2024-05-28
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MCO-957
+see-also:
+replaces:
+superseded-by:
+---
+
+# Bootc Update Path 
+
+## Summary
+
+This enhancement outlines an additional image update path added to the backend of the MCO. 
+Keeping synced with CoreOS, MCO currently utilizes a rpm-ostree-based system to engage in base image specification and layering. By adding a bootc update path, MCO, in the future, will embrace the Image Mode for RHEL and become ready to support image-based upgrade process.
+
+## Motivation
+
+Currently the MCO uses rpm-ostree as its backend for layered OS updates. The introduction of bootc will necessitate another update pathway in the MCO. In older RHCOS versions we have a “non-layered” update path, then there is the rpm-ostree path, and the newly added bootc update path. Incorporating a bootc update path into MCO also opens the door for OCP to offer more rooms for customization through configmaps and bootable containers. 
```

Comment:
Suggestion: I think it would also be good to call out alignment with image-based RHEL efforts in non-openshift environments as a key motivation